core/primitives-core/src/version.rs

```diff
@@ -120,6 +120,8 @@ pub enum ProtocolFeature {
     RejectBlocksWithOutdatedProtocolVersions,
     #[cfg(feature = "protocol_feature_simple_nightshade_v2")]
     SimpleNightshadeV2,
+    #[cfg(feature = "protocol_feature_restrict_tla")]
+    RestrictTLA,
```

Comment:
To be consistent with our codebase, please capitalize only the first letter of acronyms. `NearVmRuntime` is an example of this. This is also in line with [Rust RFC 430](https://github.com/rust-lang/rfcs/blob/188cc17ad38b201867955fb4a51c306c0704b6cf/text/0430-finalizing-naming-conventions.md).

```suggestion
    RestrictTla,
```